renderer/components/test-results/ResultRow.js

```diff
@@ -64,21 +64,29 @@ const SummaryContainer = styled(Flex)`
 const WebsitesSummary = ({anomalyCount, totalCount}) => {
   return <SummaryContainer flexWrap='wrap'>
     <Box width={1}>
-      <Text color={anomalyCount > 0 ? theme.colors.red8 : theme.colors.black}><MdClear /> {anomalyCount} blocked</Text>
+      <Text color={anomalyCount > 0 ? theme.colors.red8 : theme.colors.black}>
+        <MdClear /><FormattedMessage id='TestResults.Overview.Websites.Blocked.Plural' values={{ Count: anomalyCount }}/>
+      </Text>
     </Box>
     <Box width={1}>
-      <Text><MdWeb /> {totalCount} tested</Text>
+      <Text>
+        <MdWeb /><FormattedMessage id='TestResults.Overview.Websites.Tested.Plural' values={{ Count: totalCount }} />
+      </Text>
     </Box>
   </SummaryContainer>
 }
 
 const IMSummary = ({anomalyCount, totalCount}) => {
   return <SummaryContainer flexWrap='wrap'>
     <Box width={1}>
-      <Text color={anomalyCount > 0 ? theme.colors.red8 : theme.colors.black}><MdClear /> {anomalyCount} blocked</Text>
+      <Text color={anomalyCount > 0 ? theme.colors.red8 : theme.colors.black}>
+        <MdClear /><FormattedMessage id='TestResults.Overview.InstantMessaging.Blocked.Plural' values={{ Count: anomalyCount }}/>
+      </Text>
     </Box>
     <Box width={1}>
-      <Text><MdDone /> {totalCount} tested</Text>
+      <Text>
+        <MdDone /> <FormattedMessage id='TestResults.Overview.InstantMessaging.Available.Plural' values={{ Count: totalCount }} />
```

Comment:
@hellais On the mobile app, this part shows the number of available IMs instead of `totalCount` like in Website summary. Should we switch this also to show `totalCount - anomalyCount` here?

Reply:
Yeah let's align it to what we are doing on mobile.